src/components/Filter.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Select } from "@mui/material";
+
+const myArray: string[] = ["apple", "banana", "cherry"];
```

Comment:
I know temp stuff, but create an directory called utils.
Add a constants.ts file for stuff like this

Reply:
Also this is the directory where you can create a helper_functions.ts file

Reply:
done.